interface/modules/custom_modules/oe-module-prior-authorizations/public/index.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/*
+ *  package OpenEMR
+ *  link    https://www.open-emr.org
+ *  author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  Copyright (c) 2022.
+ *  All Rights Reserved
+ */
+
+require_once dirname(__FILE__, 5) . "/globals.php";
+require_once dirname(__FILE__, 2) . '/vendor/autoload.php';
+
+use Juggernaut\OpenEMR\Modules\PriorAuthModule\Controller\AuthorizationService;
+use Juggernaut\OpenEMR\Modules\PriorAuthModule\Controller\ListAuthorizations;
+use OpenEMR\Core\Header;
+use OpenEMR\Common\Csrf\CsrfUtils;
+
```

Comment:
assuming need some sort of acl access here

Reply:
No access restrictions were required by the user that had this built.